language/Dutch/strings.po

```diff
@@ -1597,11 +1597,11 @@ msgstr "Geluid\tOK\n"
 
 #: src/frontend/mame/ui/selgame.cpp:1079
 msgid "Capture\tUnimplemented\n"
-msgstr ""
+msgstr "Opname\tOngeïmplementeerd\n"
```

Comment:
"Niet geïmplementeerd" is beter?

Reply:
This is a bit of a gray area as it's imperfect dutch, but a lot closer to the original string. Shall I change them all or not?

Reply:
Yeah, change them

Reply:
I changed them all!

Reply:
thanks